UI/OverlayUtil.cs

```diff
@@ -0,0 +1,55 @@
+﻿using ColossalFramework.Math;
```

Comment:
I've seen somebody suggesting putting those `using`s inside the namespace. But I'm not sure why though lol.

Reply:
I think there is a reason but I forgot. this was automatic code generation which is why its like this